cloud_filter_chain:
  filter_chain:
    - name: RobotBodyFilter
      type: robot_body_filter/RobotBodyFilterPointCloud2
      params:
        frames/sensor: 'camera_frame_optical'
        filter/keep_clouds_organized: True
        filter/do_clipping: True
        filter/do_contains_test: True
        filter/do_shadow_test: False
        sensor/point_by_point: False
        sensor/min_distance: 0.0
        sensor/max_distance: 5.0
        body_model/inflation/scale: 1.07
        body_model/inflation/padding: 0.01
        body_model/robot_description_param: 'robot_description'
        transforms/buffer_length: 15.0
        transforms/timeout/reachable: 0.2
        transforms/timeout/unreachable: 0.2
        bounding_sphere/compute: False
        bounding_box/compute: False
        oriented_bounding_box/compute: False
        local_bounding_box/compute: False
        debug/pcl/inside: False
        debug/pcl/clip: False
        debug/pcl/shadow: False
        debug/marker/contains: False
        debug/marker/shadow: False
        bounding_sphere/debug: True
